features/permissions-policy.yml

```diff
@@ -0,0 +1,7 @@
+name: Permissions policy
+description: The `Permissions-Policy` response header sets whether a policy-controlled feature, such as an API, may be used in a document.
```

Comment:
The `allow` attribute is the other entrypoint to this feature and should be mentioned in the description, I think.